### PR TITLE
Theme idea previous events & using theme ideas max from the meta

### PR DESCRIPTION
**Pull request by local-minimum**
**Builds upon #1569 and implements what was previously #1570**
* UI uses the event-meta fully to count events.
* Adds event-meta to simulator so that events from the simulator can have themes suggested again.
* Lets user see last 6 event themes from same event group to help them not suggest things that just recently was the theme. This means it's not specific to LD but can be used by any event-group.
* Handles events that didn't have theme.

image

A note on previous LD. They don't need updating for this PR to work with them. There's nothing in the UI that would allow users to see the theme-idea page of an event that has completed, but it is probably good to use the new event shrub tool to set their idea-max to 3 in case of future changes to the UI.

### CODE REVIEW

**Comment by mikekasprzak** [comment]
TODO: set `theme-idea-limit` for all existing events.

File: src/shrub/tools/theme/theme-selection.php [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+	if ($ACTION) {
+		print " ** ERROR: Action '".$ACTION."' unknown.\n";
+	}
+	print "\nACTIONS:\n";
+	print "\t* 'has-selection': If event includes theme suggestions round.\n";
+	print "\t* 'idea-limit': The number of idea suggestions per user.\n";
+}
+
+print "\n";
+?>
```
Comment (mikekasprzak) [inline src/shrub/tools/theme/theme-selection.php]: Don't close PHP files that are "all code".

Also file should end with a newline.

**Comment by local-minimum** [comment]
What is blocking this merge?

**Comment by mikekasprzak** [comment]
Database backed up. Entries of `theme-idea-limit` set to `3` for nodes 10, 9405, 32802, and 49883 added.

**Comment by mikekasprzak** [comment]
Sorry for the late merge. I mistakenly thought this did more.

**Pull request merged**